src/core.c/BagHash.pm6

```diff
@@ -149,7 +145,7 @@ my class BagHash does Baggy {
                 nqp::if(
                   nqp::existskey(elems,$key),
                   nqp::getattr(nqp::atkey(elems,$key),Pair,'$!value'),
-                  0
+                  # 0 the value of existskey if the key doesn't exist
```

Comment:
Ah btw I don't know if this change was intended to be here.

Reply:
It is...   to make it similar to the approach for Bag.AT-KEY

Reply:
OK! Just looked different in an otherwise repetitive diff, so made me wonder.